packages/phalanx/src/Phalanx_KokkosDeviceTypes.hpp

```diff
@@ -16,27 +16,23 @@ namespace PHX {
 
   //#include <Kokkos_Cuda.hpp>
   typedef Kokkos::Cuda Device; 
-  typedef Kokkos::LayoutRight Layout;
   
 #elif defined(PHX_KOKKOS_DEVICE_TYPE_OPENMP)
   
   //#include <Kokkos_hwloc.hpp>
   //#include <Kokkos_OpenMP.hpp>
   typedef Kokkos::OpenMP Device; 
-  typedef Kokkos::LayoutRight Layout;
```

Comment:
It was only needed for the original version of Intrepid2 where the MDFields were passed into Intrepid2 functions in a wrapper. The wrapped needed to know layout information. Now that Intrepid2 is refactored to use DynRankViews this is no longer needed.